src/styles/block-card.css

```diff
@@ -0,0 +1,97 @@
+/* #region Card */
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  color: #060b35;
+  --button-color: #00acdc;
+  --button-height: 40px;
+}
+
+.card__box-img {
+  display: flex;
+  padding: 0 3px;
+  margin: 0 0 40px;
+}
+
+.card__img {
+  width: 100%;
+  height: 134px;
+}
+
+.card__name {
+  font-family: Roboto, sans-serif;
```

Comment:
Please move this rule to body selector. You can declare this rule once and then not write it every time for all elements. Fix for all selectors!
And also it's not only for font-family, you can also declare default value of font-weight, line-height etc. in body for prevent reuse of this rules